packages/icons/src/tools.svg

```diff
@@ -0,0 +1 @@
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M8.67 7.168a2.951 2.951 0 0 0 5.038-2.105c-.002-.379-.448-.502-.716-.234l-1.04 1.04a.5.5 0 0 1-.708 0L9.968 4.594a.5.5 0 0 1 0-.707l1.04-1.04c.268-.268.145-.714-.234-.717A2.951 2.951 0 0 0 8.67 7.167Z" stroke="#9293AB"/><path d="m7.89 5.964-4.229 4.229c-.935.935-1.695 1.41-.56 2.544 1.134 1.134 1.609.374 2.544-.56l4.229-4.23" stroke="#9293AB"/></svg>
```

Comment:
Please add currentColor instead of harcoded color

Reply:
done